Qase_Test/Src/Tests/UiTests/LoginTests.cs

```diff
@@ -0,0 +1,46 @@
+using NUnit.Framework;
+using Qase_Test.Core;
+using Qase_Test.Core.Browser.Service;
+using Qase_Test.Fakers;
+using Qase_Test.Models;
+using Qase_Test.Pages;
+using Qase_Test.Steps.UiSteps;
+using Qase_Test.Tests.Base;
+
+namespace Qase_Test.Tests.UiTests
+{
+    public class LoginTests : BaseTest
+    {
+        private LoginSteps _loginSteps;
+        private HomePage _homePage;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _loginSteps = new LoginSteps();
+            _homePage = new HomePage();
+        }
+
+        [Test]
+        public void LoginTest()
+        {
+            _loginSteps.Login(Settings.GetUser());
+
+            Assert.AreEqual(BrowsersService.GetDriver.Url, ReadProperties.HomeUrl, "HomePage didn't open");
+            Assert.True(_homePage.WaitForOpen(), "HomePage didn't open");
+        }
+
+        [Test]
+        public void LoginWithWrongPasswordTest()
```

Comment:
It is better to rename to the wrong creeds because in the test itself you and the email you enter incorrectly. Or enter the correct email.